.gitignore

```diff
@@ -22,3 +22,4 @@ dump.*
 
 server/db/seederCredentials.json
 v1doc.md
+.vscode/settings.json
```

Comment:
we are adding this vscode settings to gitignore now?

Reply:
ya it's just personal setting from me because i have different setup in every project

Reply:
ah gotcha! sounds good